examples/annotation/detect_hijacks.py

```diff
@@ -4,10 +4,17 @@
 
 from __future__ import print_function
 
+import argparse
+import datetime
 import json
 import logging
+import os
+import time
 
+from tabi.emulator import parse_registry_data
 from tabi.emulator import detect_hijacks
+from watchdog.events import FileSystemEventHandler
```

Comment:
Please add this new dependency in setup.py. May be we should create a requirements.txt?